app/src/swig/app.i

```diff
@@ -191,8 +191,7 @@ static void RegisterLibrariesHelper(
         const std::string& library = it->first;
         const std::string& version = it->second;
         firebase::App::RegisterLibrary(library.c_str(),
-                                       version.c_str(),
-                                       nullptr);
+                                       version.c_str());
```

Comment:
This was a recent change to the Firebase C++ SDK, so you might just be out of date on that.  https://github.com/firebase/firebase-cpp-sdk/pull/1217/files#diff-fa379ca24a0012efe3cc69026f1be97dcba3f5883f45c26cb5314f4b2c3190ee

Reply:
I see, we need to update subrepo too

But then, are there anyway we could also update subrepo along with the main repo?